backend/src/app/worker.clj

```diff
@@ -497,7 +497,7 @@
   [{:keys [pool entries]}]
   (db/with-atomic [conn pool]
     (doseq [{:keys [id cron]} entries]
-      (l/trace :hint "register cron task" :id id :cron (str cron))
+      (l/info :hint "register cron task" :id id :cron (str cron))
```

Comment:
For consistency it should be trace I think.

Reply:
You are right :)